python/tank/deploy/tank_commands/setup_project_params.py

```diff
@@ -1194,7 +1197,32 @@ def get_uri(self):
         :returns: string
         """
         return self._config_uri
-        
+
+    def is_local_configuration(self):
+        """
+        Returns if the configuration is local.
+
+        :returns: True if the configuration is local, False if it comes from the AppStore, GitHub or a zip file.
+        """
+        return self._config_mode == "local"
+
+    def get_pipeline_configuration_uri(self):
```

Comment:
I used the `uri` term for configuration to denote a plethora of various formats - paths on disk, web urls, ssh git urls etc. This method is different, it will always return a path on disk - right? I suggest simply calling it `get_pipeline_configuration()`.